src/constants/labels.json

```diff
@@ -33,7 +33,8 @@
     },
     "wordChoice": {
       "title": "Level 1",
-      "description": "Das richtige Wort auswählen"
+      "description": "Das richtige Wort auswählen",
+      "speaking": "Die Übung ist für dieses Modul nicht möglich (zu wenig Wörter)"
```

Comment:
oke sorry for the confusion, the name should not be `speaking`, i meant it should be a meaningful name. e.g. `errorWrongModuleSize`